cmd/minikube/cmd/config/config.go

```diff
@@ -128,12 +133,12 @@ func ReadConfig() (MinikubeConfig, error) {
 		if os.IsNotExist(err) {
 			return make(map[string]interface{}), nil
 		}
-		return nil, fmt.Errorf("Could not open file %s: %s", constants.ConfigFile, err)
+		return nil, fmt.Errorf("Cannot open file %s: %s", constants.ConfigFile, err)
```

Comment:
Can we separate the changes towards the CLI switches from the error message changes?